package.json

```diff
@@ -33,11 +33,11 @@
     "prepare": "run-s build build:css",
     "cy:test": "cd ./cypress-tests && npm run cy:run",
     "example:build": "cd ./example && npm run build",
-    "verify": "run-s lint prepare example:build build:storybook build:snapshots",
-    "test": "cross-env react-scripts test --env=jsdom --watchAll=false",
-    "test:watch": "cross-env react-scripts test --env=jsdom",
-    "build:snapshots": "cross-env react-scripts test storyshots.test --env=jsdom  --watchAll=false",
-    "test:snapshots": "cross-env react-scripts test storyshots.test --env=jsdom",
+    "verify": "run-s lint prepare example:build build:storybook test:ufx-ui",
+    "test": "cross-env react-scripts test --watchAll=false",
+    "test:watch": "cross-env react-scripts test",
+    "test:ufx-ui": "cross-env react-scripts test storyshots.test src/components/ui src/components/format src/components/Book src/components/OrderHistory --watchAll=false",
+    "test:ufx-ui:watch": "cross-env react-scripts test storyshots.test src/components/ui src/components/format src/components/Book src/components/OrderHistory",
```

Comment:
we will also have "test" script for other purposes, as in functional tests/e2e tests (like test, test:watch scripts above). so better to keep name as `test:snapshots` and then `test:ufx-ui` can be later on used to combine all these types of test.
also let us keep snapshots to run for `all` and not just comps that are ready for hf-ui

reason to keep tests for all comps:
the task is to extend tests for components that are ready/going to update for hf-ui (like add e2e tests, add tests for redux module related to that comp, we can even do subtasks for every component).
let's not suppress tests for other already developed components because they are also almost ready but just needs to be tested more. I think if we disable tests (snapshot tests in particular) for those components, future changes in ui/format can go unnoticed and those components can go out of sync.